[INFO -- 2017-02-26 23:04:42]
Starting flip_value script
version: 2.0
[INFO -- 2017-02-26 23:04:42]
Initialization strings:
[INFO -- 2017-02-26 23:04:42]
(1) target extended-remote | ssh -T mic1 gdbserver --multi -
[INFO -- 2017-02-26 23:04:42]
(2) set sysroot /opt/mpss/3.7/sysroots/k1om-mpss-linux/
[INFO -- 2017-02-26 23:04:42]
(3) set sysroot /opt/mpss/3.7/sysroots/k1om-mpss-linux/
[INFO -- 2017-02-26 23:04:42]
(4) file /micNfs/gdb-fault-injection/injector/bin/lud/lud_check
[INFO -- 2017-02-26 23:04:42]
(5) set remote exec-file /micNfs/gdb-fault-injection/injector/bin/lud/lud_check
[INFO -- 2017-02-26 23:04:42]
(6) set args -s 2048 -n 228 -i /micNfs/gdb-fault-injection/injector/bin/lud/input_2048_th_228 -g /micNfs/gdb-fault-injection/injector/bin/lud/gold_2048_th_228
[INFO -- 2017-02-26 23:04:42]
initSignal:2
[INFO -- 2017-02-26 23:04:42]
endSignal:4
[INFO -- 2017-02-26 23:04:42]
seqSignal:30
[INFO -- 2017-02-26 23:04:42]
faultMode:1
[EXCEPTION -- 2017-02-26 23:05:03]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 23:05:03]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 23:05:03]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 23:05:03]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 23:05:03]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 23:05:03]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 23:05:03]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 23:05:03]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 23:05:03]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 23:05:03]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 23:05:03]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 23:05:03]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 23:05:03]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 23:05:03]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 23:05:03]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 23:05:03]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 23:05:03]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 23:05:03]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 23:05:03]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 23:05:03]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 23:05:03]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 23:05:03]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 23:05:03]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 23:05:03]
pythonException: invalid frame
[INFO -- 2017-02-26 23:05:03]
Backtrace BEGIN:
#0  0x0000000000404c70 in L_lud_omp_123__par_loop2_2_5 () at lud_omp.c:148
#1  0x00007ffff790ba1f in ?? ()
#2  0x00007fffffffd9c0 in ?? ()
#3  0x00007ffff4ea0100 in ?? ()
#4  0x0000000000000001 in ?? ()
#5  0x0000000000000003 in ?? ()
#6  0x0000000000000000 in ?? ()

143	            for (i = 0; i < BS; i++)
144	            {
145	                for (k=0; k < BS; k++) {
146	#pragma omp simd 
147	                    for (j = 0; j < BS; j++) {
148	                        sum[j] += temp_left[BS*i + k] * temp_top[BS*k + j];
149	                    }
150	                }
151	#pragma omp simd 
152	                for (j = 0; j < BS; j++) {

Backtrace END
Memory content before bitflip:0x5d0x000x000x00
Fault Mode: Double bit-flip
base address to flip value: 0x7fffffffd90c
address max offset: 4
Memory content after  bitflip:0x5d0x180x000x00
frame name: L_lud_omp_123__par_loop2_2_5
symbol name: chunks_in_inter_row
symbol filename: lud_omp.c
symbol line: 45
value: 6237
value address: 0x7fffffffd90c
Type: The type is an integer type.
Type sizeof: 4

[INFO -- 2017-02-26 23:05:03]
Fault Injection Successful after 21.0611460209s
[INFO -- 2017-02-26 23:05:05]
event type: exit
[INFO -- 2017-02-26 23:05:05]
exit code: 0
